linux_os/guide/services/cron_and_at/restrict_at_cron_users/file_cron_allow_exists/tests/dne.fail.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
Is the name `dne.fail.sh` intentional? Maybe we can rename the file to more easily reflect the test?

Reply:
dne is acronym for "does not exist"